plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1421,10 +1421,62 @@ public void testTimestampCoercion()
                 .addRoundTrip("TIMESTAMP '1969-12-31 23:59:59.999999499999'", "TIMESTAMP '1969-12-31 23:59:59.999999'")
                 .addRoundTrip("TIMESTAMP '1969-12-31 23:59:59.9999994'", "TIMESTAMP '1969-12-31 23:59:59.999999'")
 
-                // CTAS with Trino, where the coercion is done by the connector
-                .execute(getQueryRunner(), trinoCreateAsSelect("test_timestamp_coercion"))
-                // INSERT with Trino, where the coercion is done by the engine
-                .execute(getQueryRunner(), trinoCreateAndInsert("test_timestamp_coercion"));
+                .execute(getQueryRunner(), trinoCreateAsSelect("test_timestamp_coercion"), (temporaryRelation) -> {
```

Comment:
Can we extract them as a separate test instead of having them as a part of coercion tests.

Reply:
Isn't this actually part of the coercion validation that the output type is coherent?

Reply:
Main goal of the test is to validate the type mapping and for validation involving `SHOW CREATE` we might need a dedicated tests for it. WDYT ?